django/db/models/base.py

```diff
@@ -442,7 +442,11 @@ def __init__(self, *args, **kwargs):
                 if val is _DEFERRED:
                     continue
                 _setattr(self, field.attname, val)
-                kwargs.pop(field.name, None)
+                if kwargs.pop(field.name, NOT_PROVIDED) is not NOT_PROVIDED:
+                    raise TypeError(
+                        f'{cls.__name__}() got both positional and keyword '
```

Comment:
```suggestion
                        f'{cls.__qualname__}() got both positional and keyword '
```

Reply:
Good shout. (Perhaps other similar exceptions could be updated in another PR.)